image/mk_europe_edition64.sh

```diff
@@ -73,9 +73,13 @@ cd ../../
 
 # Use latest qemu-aarch64-static version, since aarch64 doesn't seem to be that stable yet..
 if [ "$(arch)" != "aarch64" ]; then
-    wget -P mnt/usr/bin/ https://github.com/multiarch/qemu-user-static/releases/download/v7.2.0-1/qemu-aarch64-static
-    chmod +x mnt/usr/bin/qemu-aarch64-static
-    unshare -mpfu chroot mnt qemu-aarch64-static -cpu cortex-a72 /bin/bash -c /root/stratux/image/mk_europe_edition_device_setup64.sh
+    #wget -P mnt/usr/bin/ https://github.com/multiarch/qemu-user-static/releases/download/v7.2.0-1/qemu-aarch64-static
+    #chmod +x mnt/usr/bin/qemu-aarch64-static
+    #unshare -mpfu chroot mnt qemu-aarch64-static -cpu cortex-a72 /bin/bash -c /root/stratux/image/mk_europe_edition_device_setup64.sh
+    # For use on Gentoo systems with Qemu
+    cp /usr/bin/qemu-aarch64 mnt/usr/bin
+    chmod +x mnt/usr/bin/qemu-aarch64
+    unshare -mpfu chroot mnt qemu-aarch64 /bin/bash -c /root/stratux/image/mk_europe_edition_device_setup64.sh
```

Comment:
Any specific reason why you dropped the -cpu flag? Not sure why I added it in the past, but most likely I had problems without it..

Reply:
When I use the Qemu commands as written, I get this -

`+ unshare -mpfu chroot mnt qemu-aarch64-static -cpu cortex-a72 /bin/bash -c /root/stratux/image/mk_europe_edition_device_setup64.sh
/bin/bash: line 1: /root/stratux/image/mk_europe_edition_device_setup64.sh: cannot execute: required file not found`

Upon you asking the question, I rewrote it to use the inbuilt aarch64 Qemu container (my Gentoo system does not like the downloaded qemu-aarch64-static one for some reason) but left the -cpu argument in, and it actually seems to work with that argument in place, so I'll leave it in.

For the record, building without the argument seems to work without issue.

I'm redoing my fork to use a different branch at the moment. Will generate a PR once done.